examples/athena_plugin/athena_plugin/athena.py

```diff
@@ -83,7 +83,7 @@ def no_io_wf():
     inputs=kwtypes(iso_code=str),
     task_config=AthenaConfig(database="vaccinations"),
```

Comment:
btw, in order to successfully run this example on AWS I had to pass all AthenaConfig parameters
```suggestion
    task_config=AthenaConfig(workgroup="primary", catalog="AwsDataCatalog", database="vaccinations"),
```